src/site/content/en/blog/at-property/index.md

```diff
@@ -0,0 +1,188 @@
+- `custom-ident` (a custom identifier string)
+
+
+Setting a syntax allows for the custom properties we initialize to be
+interpreted as more than just strings by the browser, enabling them to be parsed
+as typed values. This has a lot of great benefits.
+
+Let's take a look at animating a gradient to illustrate this point. Currently,
+there is no way to smoothly animate (or interpolate) between gradient values, as
+each gradient declaration is parsed as a string. 
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="support1.gif" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Using a custom property with a "number" syntax, the gradient on the left shows a smooth transition between stop values. The gradient on the right uses a default custom property (no syntax defined) and shows an abrupt transition.
+  </figcaption>
+</figure>
+
+In this example, the gradient stop percentage is being animated from a starting
+value of 40% to an ending value of 100% via a hover interaction. We expect a
+smooth transition of that top gradient color downward.
+
+The browser on the right supports the Houdini Properties and Values API,
+enabling a smooth gradient stop transition, and the on the right does not. The
+non-supporting browser is only able to understand this change as a string going
+from point A to point B. There is no opportunity to interpolate the values, and
+thus you don't see that smooth transition.
+
+However, if you declare syntax type when writing custom properties, and then use
+those custom properties to enable the animation, you'll see the transition. You
+can instantiate the custom property `--gradPoint` like so:
+
+```css
+/* Check for Houdini support & register property */
+@supports (background: paint(something)) {
+  @property --gradPoint {
+    syntax: '<percentage>';
+    inherits: false;
+    initial-value: 40%;
+  }
+}
+```
+
+And then when it comes time to animate it, you can update the value from the initial `40%` to `100%`:
+
+```css
+@supports (background: paint(something)) {
+  .post:hover,
+  .post:focus {
+    --gradPoint: 100%;
+  }
+}
+```
+
+This will now enable that smooth gradient transition.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="demo.gif" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Smoothly transitioning gradient borders. <a href="https://glitch.com/~houdini-gradient-borders">See Demo on Glitch</a>
+  </figcaption>
+</figure>
+
+## Conclusion 
+
+The `@property` syntax makes an exciting technology even more accessible by
+allowing us to write semantically meaningful CSS within CSS itself. To learn
+more about CSS Houdini and the Properties and Values API, check out these
+resources:
+
+- [Is Houdini Ready Yet?](http://ishoudinireadyyet.com/)
+- [Smarter custom properties with Houdini's new API](https://web.dev/css-props-and-vals/)
+- [Houdini CSSWG Issue Queue](https://github.com/w3c/css-houdini-drafts/issues)
```

Comment:
MDN has a nice section on Houdini. https://developer.mozilla.org/en-US/docs/Web/Houdini

Reply:
Please add this link. The other links are limited or ephemeral.

Reply:
The link is added in the 2nd spot